src/popup/popup.js

```diff
@@ -316,7 +347,12 @@ async function loadStoredSearch() {
           addThumbnailsToDOM(pageData);
           pageNo = Number(pageNo) + 1;
         }
+        elements.clearSearchButton[0].removeAttribute('style');
+      } else {
+        elements.clearSearchButton[0].setAttribute('style', 'display: none');
```

Comment:
Instead of this approach, please use the one already in the code, i.e using `classList` property of the node to add and remove the already defined class `display-none`. (eg [here](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/helper.js#L147) and [here](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/helper.js#L151))